### DISCUSSION TITLE
blog/work-i-can-insist-more-time

### DISCUSSION THREAD

**Discussion by giscus** [bbruceyuan/bbruceyuan.github.io#18] [Announcements]
# blog/work-i-can-insist-more-time

我依然在工作，我依然在挣扎，我依然在挣扎着完成工作，希望新的工作策略可以帮助我再坚持坚持！！！

https://bruceyuan.com/blog/work-i-can-insist-more-time.html

**Comment by CangHaiQingYue** [comment]
哈哈哈，佬，还在字节么

**Reply by bbruceyuan** [reply]
标题就答案（🤣
